addons/payment_sips/models/payment_acquirer.py

```diff
@@ -0,0 +1,62 @@
+# coding: utf-8
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+# Copyright 2015 Eezee-It
+
+from hashlib import sha256
+
+from odoo import models, fields, api
+from odoo.tools.translate import _
+from odoo.addons.payment.models.payment_acquirer import ValidationError
+
+from .const import SIPS_SUPPORTED_CURRENCIES
+
+class AcquirerSips(models.Model):
```

Comment:
Absolute nitpicking:
```suggestion
class PaymentAcquirer(models.Model):
```
So that the class name matches the model